### ISSUE TITLE
Pointer input note

### ISSUE THREAD

**Issue by rwapp** [w3c/matf#282]
This follows comments from @cbayerlein 
(https://github.com/w3c/matf/issues/66#issuecomment-3161653417 & https://github.com/w3c/matf/issues/66#issuecomment-3184620657) in the [Keyboard Interface definition discussion](https://github.com/w3c/matf/issues/66) and discussion in [MATF November 19, 2025](https://www.w3.org/2025/11/19-matf-minutes.html).

Pointer input is not explicitly specified in WCAG in the same way keyboard input is addressed in SC 2.1.1. In my reading, WCAG implicitly assumes pointer support. However, in mobile applications, pointer input is not a given for several reasons: the primary input mechanism is the touch screen, and pointer hardware support was a later addition on iOS. It is therefore important that MATF clarifies that pointer input is expected.

There is no obvious place for such a note.

**SC 2.5.6 Concurrent Input Mechanisms (Level AAA)**
[SC 2.5.6](https://www.w3.org/WAI/WCAG22/Understanding/concurrent-input-mechanisms.html) states that input modalities should not be restricted, which would include external pointer devices. So adding a note here would therefore only need clarify that pointer input should be considered in touchscreen-based mobile applications.
However, this is AAA, so is possibly a higher threshold than is desirable. And more importantly, AAA is not, at the time of writing, covered in WCAG2ICT.

**SC 2.1.1 Keyboard (Level A)**
[SC 2.1.1 ](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html) includes the note "This does not forbid and should not discourage providing mouse input or other input methods in addition to keyboard operation." which we could potentially clarify this note in WCAG2Mobile to "supporting mouse input or other input methods is encouraged". However, this could risk straying into the normative language of WCAG. Additionally the paragraph stating "The use of MouseKeys would not satisfy this success criterion because it is not a keyboard equivalent to the application; it is a mouse equivalent" while not restricting pointer support, could be read as a contradiction.

**SC 2.5.1 Pointer Gestures (Level A)**
[SC 2.5.1](https://www.w3.org/WAI/WCAG22/Understanding/pointer-gestures.html) includes "Some people  ... may use a specialized or adapted input device such as a head pointer, eye-gaze system, or speech-controlled mouse emulator." Which makes this SC feel like a good candidate. However this SC covers only path-based gestures, and not pointer input as a whole. For example, activating a button with a trackball would not fall within its scope.
